### PR TITLE
:alembic: Added pruning information to ``QuickCell`` experiment.

### PR DESCRIPTION
**Pull request by Drewniok**
## Description

This PR updates the experiment for ``QuickCell``:

- the information on how many layouts are pruned at each pruning step is added to the results tables.
- the design of the wire and inverter is added.

## Checklist:

- [x] The pull request only contains commits that are related to it.
- [ ] I have added appropriate tests and documentation.
- [x] I have added a changelog entry.
- [ ] I have created/adjusted the Python bindings for any new or updated functionality.
- [x] I have made sure that all CI jobs on GitHub pass.
- [ ] The pull request introduces no new warnings and follows the project's style guidelines.

### CODE REVIEW

**Comment by Drewniok** [comment]
PR #594 needs to be merged first.

File: docs/changelog.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ Added
 - I/O:
     - SVG drawer for SiDB layouts
 - Experiments:
+    - Added information about how many layouts remain after each of QuickCell's pruning steps.
```
Comment (marcelwa) [inline docs/changelog.rst:22]: v0.6.6 has already been released. Please add the change under a new `Unreleased` header.

Comment (Drewniok) [reply docs/changelog.rst:30]: True! This was caused by the github merge. Should be fixed now.

File: experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp [hunk 2]
```diff
@@ -30,33 +30,55 @@ using namespace fiction;
...
+
+    const auto truth_tables =
+        std::vector<std::vector<tt>>{std::vector<tt>{create_id_tt()},  std::vector<tt>{create_not_tt()},
+                                     std::vector<tt>{create_and_tt()}, std::vector<tt>{create_nand_tt()},
+                                     std::vector<tt>{create_or_tt()},  std::vector<tt>{create_nor_tt()},
+                                     std::vector<tt>{create_xor_tt()}, std::vector<tt>{create_xnor_tt()},
+                                     std::vector<tt>{create_lt_tt()},  std::vector<tt>{create_gt_tt()},
+                                     std::vector<tt>{create_le_tt()},  std::vector<tt>{create_ge_tt()},
+                                     create_crossing_wire_tt(),        create_half_adder_tt(),
+                                     create_double_wire_tt()};
```
Comment (marcelwa) [inline experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp:58]: Could be an `std::array`

File: experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp [hunk 3]
```diff
@@ -30,33 +30,55 @@ using namespace fiction;
...
+
+    static const std::vector<std::string> gate_names = {"inv", "wire", "and", "nand", "or", "nor", "xor",      "xnor",
+                                                        "lt",  "gt",   "le",  "ge",   "cx", "ha",  "hourglass"};
```
Comment (marcelwa) [inline experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp:61]: You could merge the two into a `std::array<std::pair<std::vector<tt>, std::string>>` data type.

Comment (marcelwa) [reply experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp:61]: Analogously in the `quickcell_3_input.cpp`

**Review by marcelwa — APPROVED** [review]
If you could specify the changes in the title of this PR, we're ready to merge 🙂

**Comment by Drewniok** [comment]
> If you could specify the changes in the title of this PR, we're ready to merge 🙂

Thank you! Is this okay now?

**Pull request merged**